das/local_disk_das.go

```diff
@@ -67,9 +69,13 @@ func generateAndStoreKeys(dbPath string) (*blsSignatures.PublicKey, blsSignature
 	return &pubKey, privKey, nil
 }
 
-func NewLocalDiskDataAvailabilityService(dbPath string) (*LocalDiskDataAvailabilityService, error) {
+func NewLocalDiskDataAvailabilityService(dbPath string, signerMask uint64) (*LocalDiskDataAvailabilityService, error) {
```

Comment:
Do we need a DAS to know its own `signerMask`?  At the moment I don't think we do anything with the mask except pass it back to a caller, but the caller shouldn't trust us to provide the mask anyway.

Reply:
I made it untrusted now in the aggregator. I don't know yet if it matters for leaf DASes to know their signer mask.